core/src/test/java/org/elasticsearch/indices/stats/IndexStatsIT.java

```diff
@@ -378,7 +389,7 @@ public void testThrottleStats() throws Exception {
     }
 
     public void testSimpleStats() throws Exception {
-        assertAcked(prepareCreate("test1").setSettings("index.mapping.single_type", false));
+        assertAcked(prepareCreate("test1").setSettings("index.version.created", Version.V_5_6_0.id)); // allows for multiple types
```

Comment:
man it's like deleting 5 lines, very simple to `untangle` all the stuff I am doing above is so much duplication for not much win if at all. I thin this one I will just keep